src/terminal.rs

```diff
@@ -50,6 +52,12 @@ impl TerminalLoggerBuilder {
         self
     }
 
+    /// Sets the source code location type this logger will use.
```

Comment:
This description seems inappropriate (copy and paste mistake?).

Reply:
will do